packs/equipment-effects/effect-parry.json

```diff
@@ -16,9 +16,9 @@
             "value": 1
         },
         "publication": {
-            "license": "OGL",
-            "remaster": false,
-            "title": "Pathfinder Core Rulebook"
+            "license": "ORC",
```

Comment:
We should tidy the description to something like "You gain a +1 circumstance bonus to AC."